plugins/ruby-foreman-tasks/debian/gem.list

```diff
@@ -1,6 +1,6 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman-tasks-0.9.3.gem"
-GEMS="$GEMS https://rubygems.org/downloads/foreman-tasks-core-0.1.3.gem"
+GEMS="https://rubygems.org/downloads/foreman-tasks-0.10.0.gem"
+GEMS="$GEMS https://rubygems.org/downloads/foreman-tasks-core-0.2.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/daemons-1.2.4.gem"
```

Comment:
this seems to be in Foreman core with recent nightlies now and needs to get removed here